Tools/clinic/clinic.py

```diff
@@ -5903,6 +5905,7 @@ def run_clinic(parser: argparse.ArgumentParser, ns: argparse.Namespace) -> None:
             parser.error("can't use -o or filenames with --make")
         if not ns.srcdir:
             parser.error("--srcdir must not be empty with --make")
+        excludes = [os.path.join(ns.srcdir, f) for f in ns.exclude.split(",")]
```

Comment:
You perhaps need to normalize paths. Just in case there is a double slash or `./`.

Reply:
Hm, I thought `os.path` handled this already. I will investigate.

Reply:
It is not handled; I added `os.path.normpath`.